Foundation/NSObjCRuntime.swift

```diff
@@ -266,22 +266,143 @@ public func NSStringFromClass(_ aClass: AnyClass) -> String {
     neither stable nor human-readable.
  */
 public func NSClassFromString(_ aClassName: String) -> AnyClass? {
-    let aClassNameWithPrefix : String
-    let components = aClassName._bridgeToObjectiveC().components(separatedBy: ".")
-    
-    switch components.count {
-    case 1:
-        guard !aClassName.hasPrefix("_Tt") else {
-            NSLog("*** NSClassFromString(\(aClassName)): cannot yet decode mangled class names")
+    switch aClassName {
```

Comment:
The other problem here is that if a class had the NS dropped the name should still have the NS.

E.g.

`NSTask` got renamed to `Process`. If you ask the NSStringFromClass(Process.self) on the objc backed swift it will return "NSTask" whereas if you ask for it in the swift-corelibs-foundation version you can "Process" which is wrong.